pkgs/by-name/ri/ringfairy/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  stdenv,
+  darwin,
+}:
+rustPlatform.buildRustPackage {
+  pname = "ringfairy";
+  version = "unstable-2024-05-11";
```

Comment:
```suggestion
  version = "0-unstable-2024-05-11";
```

Reply:
It has some tagged releases but they're all marked `-alpha` so idk if those should be used or 0-

Reply:
What's the `0-` for? Yeah they are all alpha so I assumed it was some weird pre-1.0 versioning system.

Reply:
> Example: Given a project that has no tags / released versions at all, or applies versionless tags like latest or YYYY-MM-DD-Build, a commit authored on March 15, 2022 would have version = "0-unstable-2022-03-15".

https://github.com/NixOS/nixpkgs/blob/77874773a16d1c0507ee6a37c83a2da10bc2a276/pkgs/README.md?plain=1#L357

Reply:
I don't think that applies here. I actually wouldn't normally be using `unstable-` here in the first place since it *does* have actual releases, the only reason is that the commit that added a lockfile hasn't been released yet so this is one commit after the latest release.

Reply:
Then something more like
https://github.com/NixOS/nixpkgs/blob/77874773a16d1c0507ee6a37c83a2da10bc2a276/pkgs/README.md?plain=1#L353